### PR TITLE
Feature/cc 0097 profile drop down

### PR DESCRIPTION
**Pull request by GLC-coder**
Feature//cc 0097 profile drop-down component is ready to be merged into the main.
Screen Shot 2022-10-28 at 8 58 57 am

### CODE REVIEW

File: src/components/NavBar/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  onClose: () => void;
+  handleLogout: () => void;
+}
+
+const Profile = ({ isOpen, onOpen, onClose, handleLogout }: Props) => {
+  return (
+    <Menu isOpen={isOpen}>
+      <MenuButton
+        aria-label="User information"
+        bg="background.tertiary"
```
Comment (STY1997) [inline src/components/NavBar/Profile.tsx:26]: Please use the full name

File: src/__tests__/components/NavBar/Profile.test.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    );
+    const MyAccountElement = screen.getByText("My Account");
+    const MyTemplateElement = screen.getByText("My Template");
+    const MyOrderElement = screen.getByText("My Order");
+    const SignOutElement = screen.getByText("Sign Out");
+
+    expect(MyAccountElement).toBeInTheDocument();
+    expect(MyTemplateElement).toBeInTheDocument();
+    expect(MyOrderElement).toBeInTheDocument();
+    expect(SignOutElement).toBeInTheDocument();
```
Comment (tatoMa) [inline src/__tests__/components/NavBar/Profile.test.tsx:29]: The mock functions "onOpen" and "onClose" also can be tested if you like.
Use the testing library API "userEvent" to fire a hover(there is also a unhover) event and then test if the mock onOpen or onClose "toHaveBeenCalledTimes(1)"

**Review by tatoMa — COMMENTED** [review]
The size when hovering each button seems not right. There are gaps between the hover state of the button and the separator lines.
image

**Review by Double-Twelve — APPROVED** [review]
Excellent work Jason!

**Review by wwwseven — APPROVED** [review]

**Pull request merged**